css/style.css

```diff
@@ -0,0 +1,504 @@
+    line-height: 19px;
+  }
+
+  .arrow {
+    width: 73px;
+    padding-left: 10px;
+    padding-top: 3px;
+  }
+}
+
+@media (max-width: 767px) and (min-width: 375px) {
+  .header {
+    grid-template-columns: 1fr 1fr;
+    padding-top: 24px;
+    margin-bottom: 26px;
+    height: 22px;
+    width: 100%;
+  }
+
+  .header__logo-image,
+  .menu,
+  .header__user-block,
+  .header__search,
+  .header__cart,
+  .header__language-block {
+    display: none;
+  }
+
+  .header__logo-image_adaptive {
+    display: flex;
+    height: 22px;
+    width: 104px;
+    margin-left: 48px;
+    align-items: center;
+  }
+
+  .header__menu-bar {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  .menu-bar {
+    display: block;
+    width: 41px;
+    height: 17px;
+    margin-right: 47px;
+  }
+
+  .promo {
+    width: 100%;
+    height: 191px;
+    margin-bottom: 38px;
+    background-size: cover;
+    background-position: 19%;
+  }
+
+  .promo__block {
+    padding-left: 48px;
+    padding-top: 66px;
+  }
+
+  .promo__title {
+    font-size: 20px;
+    line-height: 19px;
+    padding-bottom: 18px;
+  }
+
+  .link {
+    font-size: 12px;
+    line-height: 11px;
+  }
+
+  .arrow {
+    width: 46px;
+    padding-left: 8px;
+    padding-top: 2px;
+  }
+}
+
+@media (max-width: 374px) and (min-width: 320px) {
+  .header {
+    grid-template-columns: 1fr 1fr;
+    padding-bottom: 25px;
+    padding-top: 24px;
+    margin-bottom: 0;
+    height: 22px;
+    width: 100%;
+  }
+
+  .header__logo-image,
+  .menu,
+  .header__user-block,
+  .header__search,
+  .header__cart,
+  .header__language-block {
+    display: none;
+  }
+
+  .header__logo-image_adaptive {
+    display: flex;
+    height: 22px;
+    width: 105px;
+    margin-left: 20px;
+    align-items: center;
+  }
+
+  .header__menu-bar {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  .menu-bar {
+    display: block;
+    width: 41px;
+    height: 16px;
+    margin-right: 20px;
+  }
+
+  .promo {
+    width: 100%;
+    height: 191px;
+    margin-bottom: 38px;
+    background-size: cover;
+    background-position: 23%;
+  }
+
+  .promo__block {
+    padding-left: 20px;
+    padding-top: 65px;
+  }
+
+  .promo__title {
+    font-size: 20px;
+    line-height: 20px;
+    padding-bottom: 17px;
+  }
+
+  .link {
+    font-size: 12px;
+    line-height: 11px;
+  }
+
+  .arrow {
+    width: 46px;
+    padding-left: 8px;
+    padding-top: 2px;
+  }
+}
+
+@media (max-width: 319px) {
+  .header {
+    grid-template-columns: 1fr 1fr;
+    padding-bottom: 25px;
+    padding-top: 24px;
+    margin-bottom: 0;
+    height: 22px;
+    width: 100%;
+  }
+
+  .header__logo-image,
+  .menu,
+  .header__user-block,
+  .header__search,
+  .header__cart,
+  .header__language-block {
+    display: none;
+  }
+
+  .header__logo-image_adaptive {
+    display: flex;
+    height: 22px;
+    width: 105px;
+    margin-left: 20px;
+    align-items: center;
+  }  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/chudik_css_end/pullRequest?prid=6014585)